tests/net/rpl/src/main.c

```diff
@@ -150,7 +150,7 @@ static int tester_send(struct net_if *iface, struct net_pkt *pkt)
 	DBG("pkt %p to be sent len %lu\n", pkt, net_pkt_get_len(pkt));
 
 #if 0
```

Comment:
why do we have an #if 0
make it in terms of some macro, or delete the whole thing

Reply:
I don't know the background why it is like that, I think the #if can be removed here. Will do that in another patch to make sure net_hexdump_frags() prints out things only when CONFIG_NET_DEBUG_RPL is enabled.

Reply:
Please do not remove the "#if 0" from this rpl test program. These were there on purpose as some of the tests need more TLC but there has not been time to fix/enhance them.

Reply:
I am not removing the #if 0 at line 535, there the comment specifically says it should stay and need more work. It's not the same issue.